손슬기/week1/2023.03.02./boj10156/boj10156.py

```diff
@@ -0,0 +1,5 @@
+a, b, c = map(int, input().split())
+if a*b <= c:
```

Comment:
알아보기 좋은 코드네요. 파이썬만의 장점을 살려서 
```suggestion
print(0 if a * b <= c else a * b -c)
```
이렇게 작성한는 방식도 좋다고 생각합니다!

Reply:
구경하며 지나가다가 눈에 띄어서 말씀드려요 :) 
코드리뷰 블럭을 잡으실 때 클릭한 다음 메시지를 작성하면 클릭한 라인이 가장 마지막으로 나오기 때문에, 영역을 드래그 해서 잡으시면 원하시는 구역을 정확하게 보여줘서 보는 입장에서 훨씬 알아보기 쉬울 것 같아요!

이번 경우로 말하면 
```python
if a*b <= c:
    print(0)
else:
    print(a*b-c)
```
이 범위로요!

Reply:
엇 그런 방법이 있었네요... 고맙습니다~
 혹시 블럭 수정은 못하나요 ...?

Reply:
> 엇 그런 방법이 있었네요... 고맙습니다~ 혹시 블럭 수정은 못하나요 ...?

어... 시도해본 적이 없어서 모르겠어요 :(
한번 시도해보시고 저도 알려주실 수 있을까요?

Reply:
잘 안되네요... 하핳